WoWPro_Leveling/Wrath/Neutral/WOTLK_ZulDrak.lua

```diff
@@ -168,91 +170,106 @@ A Precious Elemental Fluids|QID|12510|PRE|12507|M|40.5,65.6|N|From Hexxer Ubungo
 C Mushroom Mixer|QID|12514|N|All over the area, you need 10. Herbalists can use their tracking to make it easier.|M|41.4,57.4|US|
 C Creature Comforts|QID|12599|N|They are all over too, but can be hard to see. They look like a big thorn sticking up. Mouse over the area to find them.|M|46.9,61.4|US|
 T Creature Comforts|QID|12599|N|To Captain Grondel.|M|48.13,63.83|
 T Mushroom Mixer|QID|12514|N|To Hexxer Ubungo.|M|40.61,65.59|
 A Too Much of a Good Thing|QID|12516|PRE|12514|M|40.5,65.6|N|From Hexxer Ubungo.|
-T Pa'Troll|QID|12596|M|40.3,66.6|N|To Commander Kunz.|
-C Too Much of a Good Thing|QID|12516|N|Head back North to the Altar of Sseratus. The big guy you avoided earlier. Weaken him with the Mojo and give him a whuppin'.|U|38332|M|40.52,42.77|
 T Gluttonous Lurkers|QID|12527|N|Inside the building at Heb'Valok, again. To Alchemist Finklestein.|M|35.08,52.09|
+T Pa'Troll|QID|12596|M|40.3,66.6|N|To Commander Kunz.|
+C Too Much of a Good Thing|QID|12516|N|Head back North to the Altar of Sseratus. The big guy you avoided earlier. Weaken him with the Mojo and give him a whuppin'.|U|38332|M|40.52,42.77|T|Prophet of Sseratus|
+H The Argent Stand|ACTIVE|12516|M|40.78,66.29|N|Hearth or run back to The Argent Strand to turn in your quest.|
 T Too Much of a Good Thing|QID|12516|N|From Hexxer Ubungo.|M|40.61,65.59|
 A To the Witch Doctor|QID|12623|PRE|12516|M|40.5,65.6|N|From Hexxer Ubungo.|
-N End of the Argent Stand Quests.|QID|12623|
-T To the Witch Doctor|QID|12623|N|Go up the stairs next to the Amphitheater of Anguish. Zim'Torga is just ahead on your right. To Witch Doctor Khufu.|M|59.52,57.98|
+T To the Witch Doctor|QID|12623|N|Go up the stairs next to the Amphitheater of Anguish. Zim'Torga is just ahead on your right. To Witch Doctor Khufu.|M|50.20,66.36;59.52,57.98|CS|
 A Breaking Through Jin'Alai|QID|12627|PRE|12623|M|59.5,58.1|N|From Witch Doctor Khufu.|
 A The Blessing of Zim'Torga|QID|12615|M|59.5,58.1|N|From Witch Doctor Khufu.|
+f Zim'Torga|ACTIVE|12615|M|60.02,56.74|N|Get the flightpoint from Maaka.|
+h Zim'Torga|ACTIVE|12615|M|59.34,57.31|N|At Pan'ya.|T|Pan'ya|
 T The Blessing of Zim'Torga|QID|12615|N|Big statue in the middle.|M|59.4,57.2|
 A The Leaders at Jin'Alai|QID|12622|M|59.2,56.2|N|From Scalper Ahunae.|
-C The Leaders at Jin'Alai|QID|12622|N|Kill Trolls, you must kill trolls to get the bosses to spawn (Also check to see if one is up). Pay attention to you chat log, when the "bosses" yell, find him by his Totem. The items can be looted by everyone, so don't worry if someone else tags the mob first. Just don't forget to loot, they spawn in order, so it could be awhile if you miss one.|S|
-C Breaking Through Jin'Alai|QID|12627|N|Just South, Click on the Purple cauldron.|M|57.62,61.72|QO|3|
+C The Leaders at Jin'Alai|QID|12622|M|57.34,63.52|N|Kill all the trolls in the area to get the named trolls to spawn.|S|
+l Kutube'sa|ACTIVE|12622|QO|1|M|57.46,63.75|T|Kutube'sa|N|Kill Kutube'sa, and loot his treasure.|S|
```

Comment:
It isn't a K step.. you have to kill the guy for the box to be on the ground to loot. Made it l steps because if you run up after someone else has killed it you can just loot the item from the box. It isn't looted from the body.

Reply:
Cool